### ISSUE TITLE
Upgrade to Keycloak 26 base theme for new authChecker.js

### ISSUE THREAD

**Issue by ofairfoul** [keycloakify/keycloakify#739]
authChecker.js was recently [updated](https://github.com/keycloak/keycloak/commit/7acb30269bedc32e559ab28c0a5adf8d2195a378) in Keycloak 26. 
Keycloakify is currently based on [25.0.4 base theme](https://github.com/keycloakify/keycloakify/blob/main/scripts/shared/downloadKeycloakDefaultTheme.ts#L10). This project needs updating to the latest base theme to pick up these improvements.

**Comment by garronej** [comment]
Yes, 
Related to https://github.com/keycloakify/keycloakify/discussions/729

**Comment by ffalcinelli** [comment]
@garronej yes, It's related to the V2, but still it's used also on the keycloak V1 theme. 

On iPhone may cause problems without the fixes reported here: https://github.com/keycloak/keycloak/pull/35408/commits/522d801e9c0015bcfd23885af5d8eca1cddd1b5a

That resolves https://github.com/keycloak/keycloak/issues/33071

I'm currently trying to patch the resources using npx patch-package couldn't it just be updated from keycloakify directly?

**Comment by sadadash** [comment]
What is latest update over this ? We can still see old authcheck.js
